integrations/github/highlights.scm

```diff
@@ -0,0 +1,214 @@
+; ; Highlighting for TLA+ files on GitHub
```

Comment:
Just a heads up that this is more useful than just for GitHub—the same queries can also integrate with editor packages, like [`nvim-treesitter`](https://github.com/nvim-treesitter/nvim-treesitter) and [`emacs-tree-sitter`](https://emacs-tree-sitter.github.io).

Reply:
Oh yeah we have separate queries for those, since they use a different/expanded set of capture names.